data/json/player_activities.json

```diff
@@ -245,6 +280,7 @@
   {
     "id": "ACT_FERTILIZE_PLOT",
     "type": "activity_type",
+    "activity_level": "NO_EXERCISE",
```

Comment:
Fertilizing soil isn't just spreading over the topsoil (I mean you could but it's largely wasted/becomes runoff); you generally need to mix fertilizer into the soil by hand or tool (like a tine cultivator).

Probably a LIGHT_EXERCISE activity for a single plant (or very small garden), but doing manually for a field should make this more of a MODERATE_EXERCISE.